receiver/opencensusreceiver/ocmetrics/opencensus_test.go

```diff
@@ -343,17 +342,12 @@ func ocReceiverOnGRPCServer(t *testing.T, sr consumer.MetricsConsumer) (oci *Rec
 		}
 	}
 
-	_, port, err = hostPortFromAddr(ln.Addr())
+	_, port, err = testutils.HostPortFromAddr(ln.Addr())
 	if err != nil {
 		done()
 		t.Fatalf("Failed to parse host:port from listener address: %s error: %v", ln.Addr(), err)
 	}
 
-	if err != nil {
```

Comment:
This is a dead code, because we have the same condition above which runs `t.Fatalf`, which invokes `os.Exit`